sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/cloud/actuate/eventhub/EventHubHealthIndicator.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.actuate.eventhub;
+
+import com.azure.messaging.eventhubs.EventHubConsumerAsyncClient;
+import com.azure.messaging.eventhubs.EventHubProducerAsyncClient;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+
+import java.time.Duration;
+
+/**
+ * TODO (xiada) Kafka doesn't have such health indicator in Spring Boot, do we need this one?
```

Comment:
i did a little search and here is why there is not kafak health indicator https://github.com/spring-projects/spring-boot/issues/14088